keys.txt

```diff
@@ -22,5 +22,6 @@
 "Toggle fast-forward" 1073741881
 "Fleet: Fight my target" 102
 "Fleet: Gather around me" 103
-"Fleet: Hold position" 104
+"Fleet: Hold fire" 120
+"Fleet: Stay in position" 104
```

Comment:
Why rename `Hold position`?
`Hold fire` seems like it should go near `Fight my target`.
```suggestion
"Fleet: Hold fire" 120
"Fleet: Gather around me" 103
"Fleet: Hold position" 104
```

Reply:
This used to change the value for hold position so I assume thats why